<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Board Information</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="exitBu">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Management</string>
   </property>
   <widget class="QLineEdit" name="passwordEd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Pass</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Board Info</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Model Number Info</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirmwareRevisionEd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Firmware Revision</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getFromBoardBu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get from Board</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Manufacture Date (DDMMYYYY)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UnitSerialEd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ModelNumberEd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ManufactureDateEd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToBoardBu">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to Board</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LaserSerialEd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Laser Serial Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>PCB Number (First 4-digits)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PCBNumberEd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Unit Serial Number</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBu</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
